src/emcpy/io/netCDF.py

```diff
@@ -16,13 +16,11 @@ def variable_exist(filename, variable_name):
 
     Parameters
     ----------
-    `filename` : netCDF filename
-
-    `variable_name` : variable name to check within `filename`
-
+        filename : netCDF filename
+        variable_name : variable name to check within filename
     Returns
     -------
-    `result` : `True` or `False` if variable is present
+        result : True or False if variable is present
```

Comment:
While we are being "thorough", I think I actually like doing things like `result` : `True` because it highlights it as a variable in the generated documentation:  See: https://noaa-emc.github.io/emcpy/emcpy/io/netCDF.html vs https://noaa-emc.github.io/emcpy/emcpy/calcs/units.html

Thoughts?

Reply:
yes.  I like it too. But, I just find it too much typing and there is bound to be errors.
I think we need to move on from `pdoc` to `sphinx` or something else to be similar to numpy looking documentation.